api/domain/common/messages.proto

```diff
@@ -17,16 +17,19 @@ syntax = "proto3";
 // This file follows google's gRPC naming conventions:
 // https://cloud.google.com/apis/design/naming_convention
 
+import "validate/validate.proto";
+
 option go_package = "github.com/finleap-connect/monoskope/pkg/api/domain/common";
 
 package common;
 
 // ServiceInformation are the version and commit hash of a service
 message ServiceInformation {
   // version
+  // TODO: is this just a number?
```

Comment:
We can skip validating this. This is only ever sent by the server to the client when it asks for the version of m8.